### PR TITLE
Make cover-sheet branch that has no code

### LINKED ISSUE #4: Move to 1 minor release per Julia release
If we changed so each minor release was tilde bound to a given Julia release we could delete all the `if VERSION`.

 Because this accesses so many Pkg internals, we really can't claim to support `^1.4`.
It will definitely break in 1.8.

If we have a version for each release then whenever a new one comes out we cut a new branch e.g. `release-1.9` from `release-1.8`.
And we match our major and minor against Julia (keeping patch free).

Would probably change the default branch to something with no code but a readme and instructions to go see the branchs for the codes.

No point doing this for old versions that work right now 
So for now the code can stay where is.
But for 1.7 and 1.8 I think it is probably worth doing.

### PR DESCRIPTION
**Pull request by oxinabox**
This will be the last step to close #4.

When approved I will not merge this branch.
Instead I will delete the `main` branch, and will change the default Branch in github settings to `COVER`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,4 +28,33 @@ TestEnv.activate("Example") do
...
+We do this because TestEnv.jl accesses a whole ton of interals of [Pkg](https://github.com/JuliaLang/Pkg.jl).
+These internals change basically every single release.
+Maintaining compatibility in a single branch for multiple julia versions leads to code that is a nightmare.
+As such, we instead maintain 1 branch per julia minor version.
+And we tag releases off that branch with major and minor versions matching the julia version supported, but with patch versions allowed to change freely.
+
+ - [release-1.0](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.0) contains the code to support julia v1.0.x
+ - [release-1.1](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.1) contains the code to support julia v1.1.x
+ - release-1.2 does not exist yet, as we do not yet support julia 1.2
+ - release-1.3 does not exist yet, as we do not yet support julia 1.3
```
Comment (oxinabox) [inline README.md:44]:
```suggestion
 - [release-1.3](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.3) contains the code to support julia v1.3.x
```

File: README.md [hunk 2]
```diff
@@ -30,4 +28,33 @@ TestEnv.activate("Example") do
...
+One per minor release: `release-1.0`, `release-1.1` etc.
+We do this because TestEnv.jl accesses a whole ton of interals of [Pkg](https://github.com/JuliaLang/Pkg.jl).
+These internals change basically every single release.
+Maintaining compatibility in a single branch for multiple julia versions leads to code that is a nightmare.
+As such, we instead maintain 1 branch per julia minor version.
+And we tag releases off that branch with major and minor versions matching the julia version supported, but with patch versions allowed to change freely.
+
+ - [release-1.0](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.0) contains the code to support julia v1.0.x
+ - [release-1.1](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.1) contains the code to support julia v1.1.x
+ - release-1.2 does not exist yet, as we do not yet support julia 1.2
```
Comment (mmiller-max) [inline README.md:43]: Think you've done this now!

**Review by mmiller-max — APPROVED** [review]
Looks good. Might be worth some explicit instructions for how to install each version (could be on the relevant branch), e.g.

```
# For 1.4
pkg> add TestEnv@1.4
```
Or maybe a table detailing which versions of the package to add for which Julia version

**Comment by mmiller-max** [comment]
Also is this a typo

```
Consider this package has as a test-only dependency of **ChainRulesCore.jl**.
```

`has` isn't needed I think

File: README.md (hunk 2 above)
Comment (oxinabox) [reply README.md:43]:
```suggestion
 - [release-1.2](https://github.com/JuliaTesting/TestEnv.jl/tree/release-1.2) contains the code to support julia v1.2.x
```

**Pull request closed without merging**